client/cli/src/params/pruning_params.rs

```diff
@@ -57,8 +58,9 @@ impl PruningParams {
 	/// Get the block pruning value from the parameters
 	pub fn blocks_pruning(&self) -> error::Result<BlocksPruning> {
 		Ok(match self.blocks_pruning {
+			Some(0) => BlocksPruning::KeepAll,
```

Comment:
Okay that is relative random. That this switches to `KeepAll` when you give it `0`.

Maybe we should not expose `KeepAll` at all. Or we really need to introduce some new cli flag.

Reply:
what do you think about that change `blocks_pruning` to be like `state_pruning` : use the string `keep-all` for `BlocksPruning::KeepAll` and `keep-finalized`(and default) for `BlocksPruning::KeepFinalized` ? 
And IMO ,It may help to keep the style consistent with `state_pruning` and be better to be understood .

Reply:
Using same naming would be good IMO.

Reply:
> Using same naming would be good IMO.

That also be OK .

Let's change `blocks_pruning` to be like `state_pruning` : use the string `archive` for `BlocksPruning::KeepAll` and `archive-canonical`(and default) for `BlocksPruning::KeepFinalized` ?  @bkchr @arkpar

Reply:
And then a number of `BlocksPruning::Some(x)`?

Sounds good.